packages/jellyfish-transaction/src/buffer/buffer_composer.ts

```diff
@@ -438,4 +439,25 @@ export abstract class ComposableBuffer<T> implements BufferComposer {
       }
     }
   }
+
+  static mask (
+    getter: () => boolean[],
+    setter: (data: boolean[]) => void
+  ): BufferComposer {
+    return {
+      fromBuffer: (buffer: SmartBuffer): void => {
+        const num = buffer.readInt8()
```

Comment:
I think this should be un-signed.

Write a test for mask in `buffer_composer.test.ts`

Reply:
you should assert edge cases like max number `255`

Reply:
added:
https://github.com/DeFiCh/jellyfish/blob/c2b4260fb6ca410b36c4cb3a29f5e8022d0df213/packages/jellyfish-transaction/__tests__/buffer/buffer_composer.test.ts#L1038